extension/popup.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css?family=Lato');
```

Comment:
From what I understand, I think this redundant as there is already the `<link>` tag doing this in the `popup.html` head. The browser can download the `<link>` tag in parallel with other resources, while it cannot from a CSS `@import`, hence I think delete this and leave the one in the HTML. 

Although, TBH, not sure if we should use a new font just for the popup unless we use it other places... will discuss more